### PR TITLE
Add snmptrap sidecar

### LINKED ISSUE #26: Adding snmptrapd to docker-compose and making it work?
How can I add SNMP trap handling to this docker compose config like this article is detailing https://docs.librenms.org/Extensions/SNMP-Trap-Handler/ ? I am not sure, but assuming this article is meant for non-docker installation/setups.

### PR DESCRIPTION
**Pull request by Zoruk**
Fixed #26 
Same as #133 with additional fix

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -95,7 +95,9 @@ ENV S6_BEHAVIOUR_IF_STAGE2_FAILS="2" \
   LIBRENMS_DOCKER="1" \
   TZ="UTC" \
   PUID="1000" \
-  PGID="1000"
+  PGID="1000" \
+  MIBDIRS="/opt/librenms/mibs:/opt/librenms/mibs/cisco" \
```
Comment (crazy-max) [inline Dockerfile:99]: I think `MIBDIRS` should be outsourced from the Dockerfile and placed in your `08-svc-snmptrapd.sh`. We should also allow users to provide their own MIBs. `/data/mibs` could be a good place for this.

Comment (crazy-max) [reply Dockerfile:99]: Also maybe allow additional dirs with an env var like `ADDITIONAL_MIBS` and append them to `MIBDIRS`.

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Comment by dkmcgowan** [comment]
Look forward to this getting merged in and saving me the time to figure it out, been wanting to add snmp traps to my setup using docker

**Comment by crazy-max** [comment]
Closed by #180

**Pull request closed without merging**